explore_utils.py

```diff
@@ -120,3 +120,27 @@ def find_transaction_by_region(data):
     new_df = avg[avg['rev']>0]
     new_df.columns = ['Transaction']
     return new_df.sort_values(by=['Transaction'])
+
+
+def find_return_visit_stats(dataset):
+    """Find the statistics of total transactions for returnining visitors
+
+    args:
+        dataset (Dataset): the google analytics dataset.
+
+    returns:    
+        Dataframe of transaction statistics for first time visitors versus return visitors
+    """
+
+    train_df = dataset.train.copy()
+    train_df['revenue'] = train_df['totals.transactionRevenue'].astype(float).fillna(0) / 10000
+    group_df = (train_df[['fullVisitorId', 'visitNumber', 'revenue']]
+          .groupby('fullVisitorId', as_index=False)
+          .agg({'visitNumber': 'max', 'revenue': 'sum'}))
+    first_stats = (group_df[group_df['visitNumber'] == 1]['revenue']
+                  .describe(percentiles=[.95, .99, .999, .9999])
+                  .rename('First Time Visitor'))
+    return_stats = (group_df[group_df['visitNumber'] != 1]['revenue']
+                    .describe(percentiles=[.95, .99, .999, .9999])
+                    .rename('Return Visitor'))
+    return pd.concat([first_stats, return_stats], axis=1)
```

Comment:
perhaps add a line that gets rid of the extra decimal points. The counts can be changed to integers while the other statistics should be integers or have 2 decimal points.

Reply:
It's probably easier to do the reformatting in `explore.py` when you print the numbers out. Also, this way the function doesn't decide how to round, instead the user decides how many decimal points to round.

Reply:
Added a rounding before printing in `explore.py` so the accuracy of the statistics can be preserved.

Reply:
Looks great!